utils/scanrepository.go

```diff
@@ -0,0 +1,50 @@
+package utils
+
+import (
+	"fmt"
+	utils2 "github.com/jfrog/build-info-go/utils"
```

Comment:
Keeping the packages names clear
```suggestion
	biutils "github.com/jfrog/build-info-go/utils"
```